### PR TITLE
deployment configurations

### PR DESCRIPTION
**Pull request by simpslandyy**
`docker-compose down -v` : Will stop/remove containers and volumes
`docker-compose build rails-app` : Will build the rails-app using the instructions specified in the Dockerfile
- by default, RAILS_ENV=development
- you can safely ignore `warning fsevents@1.1.1: The platform "linux" is incompatible with this module.`

**IMPORTANT BEFORE YOU DOCKER-COMPOSE UP**
If you want to run in production  `cp prod.env.default .env` otherwise, `cp dev.env.default .env`

** THEN!!!*
`docker-compose up` : Will run the docker-compose.yml specified in the COMPOSE_FILE (that's specified in the .env)

It's important to check that, .css and .js files are not being served by `localhost:8080`. (Check the network tab, look for 404s, 505s, and anything talking about localhost)

This process should do uglifying, precompiling css etc etc.

### CODE REVIEW

**Comment by jmzaleski** [comment]
is there some matching prose that should go in  the readme?

**Comment by simpslandyy** [comment]
@jmzaleski I just modified the readme not too long ago in the other PR to reflect the changes in this PR :) You can go take a look!

File: Dockerfile [hunk 1]
```diff
@@ -15,13 +19,22 @@ RUN apk add --no-cache curl && \
...
+fi
 
 COPY package.json /srv/app/
 COPY yarn.lock /srv/app/
 RUN yarn install
 
 COPY . /srv/app/
 
+RUN if [ ${RAILS_ENV} = 'production' ]; then \
+bundle exec rake webpacker:compile; \
```
Comment (freeatnet) [inline Dockerfile:36]: ✅

**Review by freeatnet — APPROVED** [review]

File: dev.env.default [hunk 2]
```diff
@@ -1,5 +1,6 @@
 # Environment variables needed for running this project
 # copy this file to .env in this directory for it to be picked up by docker-compose
+RAILS_ENV=development
```
Comment (freeatnet) [inline dev.env.default:3]: Oops. `COMPOSE_FILE=docker-compose.yml:docker-compose.dev.yml` got left out here :(

Comment (simpslandyy) [reply dev.env.default:3]: I'll fix it right away.

**Pull request merged**